docs/hazmat/primitives/interfaces.rst

```diff
@@ -696,6 +696,115 @@ Key derivation functions
             the provided signature does not match the expected signature.
 
 
+Diffie-Hellman key exchange
+---------------------------
+
+.. class:: DHParameters
+
+    .. versionadded:: 0.8
+
+
+.. class:: DHParametersWithNumbers
+
+    .. versionadded:: 0.8
+
+    Inherits from :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParameters`.
+
+    .. method:: parameter_numbers()
+
+        Return the numbers that make up this set of parameters.
+
+        :return: A :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParameterNumbers`.
+
+
+.. class:: DHPrivateKey
+
+    .. versionadded:: 0.8
+
+    .. attribute:: key_size()
```

Comment:
In #1478 we're having a discussion about our use of `key_size` where better descriptors would be appropriate. Is key size the right thing to use here?